libs/sdk-ui-kit/src/typings/accessibility.ts

```diff
@@ -9,3 +9,23 @@ export interface IAccessibilityConfigBase {
     ariaDescribedBy?: React.AriaAttributes["aria-describedby"];
     role?: React.HTMLAttributes<HTMLElement>["role"];
 }
+
+/**
+ * @internal
+ */
+export interface IMenuAccessibilityConfig extends IAccessibilityConfigBase {
+    id?: string;
+    role?: "menu" | "menuitem" | "separator" | "presentation";
+    ariaDisabled?: "true" | "false";
+    // to support submenu
+    ariaHaspopup?: "true" | "false" | "menu" | "listbox" | "tree" | "grid" | "dialog";
```

Comment:
I know its original HTML prop, but it would be better to name it `ariaHasPopup`, wdyt?

Reply:
I would like to keep it as similar as possible to original